tutorials/monitoring/2-setting-up-environment/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Monitoring NServiceBus solutions: Setting up monitoring environment"
+reviewed: 2017-10-10
+summary: Setting up the Particular Service Platform to monitor an NServiceBus system.
+extensions:
+- !!tutorial
+  nextText: "Next Lesson: Configuring endpoints"
+  nextUrl: tutorials/monitoring/3-configuring-endpoints
+---
+
+include: monitoring-intro-paragraph
+
+include: monitoring-sample-solution
+
+This second lesson guides your through setting up all of the server side components that are used to monitor an NServiceBus system.
+
+
+## Setting up
+
+In order to configure an environment for monitoring with the Particular Service Platform you will need to install and configure the components in the order listed.
+
+
+### Install ServiceControl
+
+ServiceControl is a suite of back-end tools that collect useful information about your running system. The main installer includes a desktop utility called the ServiceControl Managemeny Utility which can be used to create an manage ServiceControl and Monitoring instances.
+
+Download and run the latest ServiceControl installer.
+
+SCREENSHOT - ServiceControl Installer
+
+Once it has finished installing, run the ServiceControl Management Utility.
+
+
+### Create ServiceControl instance
+
+A ServiceControl instance collects messages from the error queue and manages retrying them. It also collects messages from the audit queue. 
+
+In the ServiceControl Management Utility, click **+ NEW INSTANCE** and select **ServiceControl instance** from the list of options.
+
```

Comment:
I would add a NOTE here that guides users that have already run out of Trial License (Standard License users). I would emphasize that getting license is easy :). See: https://github.com/Particular/LaunchWave.DevOpsAdvancedMonitoring/issues/266